Kvent/models.py

```diff
@@ -25,6 +25,7 @@ def get_address(self):
       
 myDate = datetime.now()
 formatedDate = myDate.strftime("%Y-%m-%d %H:%M:%S")
+
```

Comment:
you should uncomment date_time and participants because you use it in two methods below.

Reply:
already fixed it. Thanks!